source/_integrations/frontier_silicon.markdown

```diff
@@ -93,15 +72,16 @@ Overview of the info dialog:
 
 ## Development
 
-Support is provided through the Python [fsapi] module. The Python module was developed by using the documentation provided by [flammy] and
+Support is provided through the Python [afsapi] module. The Python module was developed by using the documentation provided by [flammy] and
 is based on [tiwillam]'s fsapi project. Special thanks to both developers, this integration would have not been possible without their work.
```

Comment:
I think this section just needs to go. It isn't relevant for the end-user of Home Assistant.